src/shared/ui/SelectTabs/SelectTabs.jsx

```diff
@@ -0,0 +1,34 @@
+import "./SelectTabs.css";
+import cx from "classnames";
+
+const SelectTabs = ({ genres = [], selectedGenre, onSelect }) => {
+  return (
+    <div className="selector">
+      {genres.map((genre) => {
```

Comment:
`genres.map` can be undefined, because it's possible that we get an empty array  `genres`, so it's better to check undefined possibility using the operator `?`

`genres?.map`

please, take a look https://learn.javascript.ru/optional-chaining 

<img width="591" alt="image" src="https://github.com/AnnaMartis/Epam-project/assets/36797396/d603787e-c934-49ec-804a-1327e78b6048">


<img width="689" alt="image" src="https://github.com/AnnaMartis/Epam-project/assets/36797396/02b4d6e0-5029-4cc8-961d-6b8ed37efea2">

same for other cases

Reply:
The reason I didn't use optional chaining was because I had put the default value of genres empty array -  [ ],

const SelectTabs = ({ genres = [ ], selectedGenre, onSelect }) => {....

Reply:
but maybe it is better to add optional chaining, because the prop argument can be null

Reply:
yeah, agree, an empty array is okay for `map` but `null` is not, good point